src/utils/getters/getEventHeight.js

```diff
@@ -0,0 +1,6 @@
+export function getEventHeight(tableCell, startDate, endDate) {
```

Comment:
Looks to me, like you need this value to hardcode your cell style or something. Maybe it's possible to write some css to make the "event" responsive ?